src/power_grid_model_io/converters/vision_excel_converter.py

```diff
@@ -37,16 +37,21 @@ def __init__(
         self,
         source_file: Optional[Union[Path, str]] = None,
         language: str = "en",
+        terms_changed: Optional[dict] = None,
         mapping_file: Optional[Path] = None,
         log_level: int = logging.INFO,
-    ):
+    ):  # pylint: disable=too-many-arguments
         _mapping_file = _mapping_file = (
             mapping_file if mapping_file is not None else Path(str(DEFAULT_MAPPING_FILE).format(language=language))
         )
```

Comment:
cfr. my above suggestion + redundant assignment
```suggestion
        _mapping_file = Path(
            mapping_file if mapping_file is not None else str(DEFAULT_MAPPING_FILE).format(language=language)
        )
```

Reply:
This is not part of this PR but indeed could use the suggested rework